sites/kit.svelte.dev/src/lib/search/search.js

```diff
@@ -70,11 +75,15 @@ export function search(query) {
 	return results;
 }
 
-/** @param {string} href */
-export function lookup(href) {
-	return map.get(href);
+/** @param {number} i */
+export function lookup(i) {
+	return map.get(i);
```

Comment:
this broke 'recent searches', which is driven by `localStorage.recent_searches`.  It can't really be a number, because any changes to the docs structure would cause things to get out of sync

Reply:
ok. I reverted that